app/constants/order-sorting-options.js

```diff
@@ -0,0 +1,44 @@
+export const ORDER_SORTING_OPTIONS = [
```

Comment:
maybe rename the constant (and file) to `BOX_CONTENT_SORTING_OPTIONS`
Because you're not actually sorting orders here